src/wp-includes/ms-functions.php

```diff
@@ -1856,7 +1856,7 @@ function check_upload_mimes( $mimes ) {
 	$site_mimes = array();
 	foreach ( $site_exts as $ext ) {
 		foreach ( $mimes as $ext_pattern => $mime ) {
-			if ( '' !== $ext && false !== strpos( $ext_pattern, $ext ) ) {
+			if ( '' !== $ext && preg_match( '!^(' . $ext_pattern . ')$!i', $ext ) ) {
```

Comment:
Super minor nitpick: Caould we use the `/` delimiter? This might look like a personal preference, and I don't think WordPress has a CS guideline for this, but the forward slash seems to be the [most widely](https://github.com/WordPress/wordpress-develop/search?q=preg_match) used delimiter.

Reply:
Sure, I don't have a specific preference for this one. Since `/` is used a couple of times I am ok with that as well.

```suggestion
			if ( '' !== $ext && preg_match( '/^(' . $ext_pattern . ')$/i', $ext ) ) {
```